samza-kafka/src/main/java/org/apache/samza/system/kafka/KafkaSystemAdmin.java

```diff
@@ -564,7 +535,7 @@ public boolean clearStream(StreamSpec streamSpec) {
    * @param spec a StreamSpec object
    * @return KafkaStreamSpec object
    */
-  KafkaStreamSpec toKafkaSpec(StreamSpec spec) {
+  public KafkaStreamSpec toKafkaSpec(StreamSpec spec) {
```

Comment:
why do we need this to be public?

Reply:
this method may be used from a different package.